yarn.lock

```diff
@@ -1040,10 +1040,10 @@
     probe.gl "3.0.1"
     puppeteer "1.8.0"
 
-"@streetscape.gl/monochrome@^1.0.0-beta.1":
-  version "1.0.0-beta.2"
-  resolved "https://registry.yarnpkg.com/@streetscape.gl/monochrome/-/monochrome-1.0.0-beta.2.tgz#f5df7f81a5047eb9c6fab7ffb6533e58643446c6"
-  integrity sha512-8lrbew2O1NPOL/ojZSxm77c2pm/XqbYDBsGbmkE7xY9ZWMwxkdvG8YD1dXzTBAKNvPAu06c6db1oLsDzlUWxXA==
+"@streetscape.gl/monochrome@^1.0.0-beta.3":
+  version "1.0.0-beta.3"
+  resolved "https://registry.npmjs.org/@streetscape.gl/monochrome/-/monochrome-1.0.0-beta.3.tgz#307d6ad85341313bcb73c76f7124384826453509"
```

Comment:
You don't have the right registry in you `.npmrc`

Make sure that you have this line in your .npmrc:
```
registry=https://registry.yarnpkg.com
```